src/envo/envo-edit.owl

```diff
@@ -40370,6 +40376,96 @@ AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#inSubset> <htt
+AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#inSubset> <http://purl.obolibrary.org/obo/ENVO_06105116> "envoPlastics")
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> <http://www.gesamp.org/publications/guidelines-for-the-monitoring-and-assessment-of-plastic-litter-in-the-ocean>) rdfs:comment <http://purl.obolibrary.org/obo/ENVO_06105116> "In the literature, including GESAMP Report #99, this term is often referenced as part of abandoned, lost or otherwise discarded fishing gear (ALDFG)."@en)
+AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/ENVO_06105116> "discarded fishing gear"@en)
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_06105116> <http://purl.obolibrary.org/obo/ENVO_06105115>)
+
+# Class: <http://purl.obolibrary.org/obo/ENVO_06105117> (abandoned fishing gear)
+
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> <https://www.merriam-webster.com/dictionary/abandoned>) <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_06105117> "Fishing gear which has been purposefully left by the owner."@en)
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/contributor> <http://purl.obolibrary.org/obo/ENVO_06105117> <https://orcid.org/0000-0002-2996-719X>)
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/contributor> <http://purl.obolibrary.org/obo/ENVO_06105117> <https://orcid.org/0000-0002-8166-0486>)
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/creator> <http://purl.obolibrary.org/obo/ENVO_06105117> <http://orcid.org/0000-0003-1072-8081>)
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/date> <http://purl.obolibrary.org/obo/ENVO_06105117> "2021-05-07T18:11:07Z"^^xsd:dateTime)
+AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#inSubset> <http://purl.obolibrary.org/obo/ENVO_06105117> "envoPlastics")
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> <http://www.gesamp.org/publications/guidelines-for-the-monitoring-and-assessment-of-plastic-litter-in-the-ocean>) rdfs:comment <http://purl.obolibrary.org/obo/ENVO_06105117> "In the literature, including GESAMP Report #99, this term is often referenced as part of abandoned, lost or otherwise discarded fishing gear (ALDFG)."@en)
+AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/ENVO_06105117> "abandoned fishing gear"@en)
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_06105117> <http://purl.obolibrary.org/obo/ENVO_06105115>)
+
+# Class: <http://purl.obolibrary.org/obo/ENVO_06105118> (lost fishing gear)
+
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> <https://www.oxfordlearnersdictionaries.com/definition/english/lost_1?q=lost>) <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_06105118> "Fishing gear which is unable to be found and remains in the environment."@en)
```

Comment:
>Fishing gear which is unable to be found and remains in the environment.

This is more a definition of unfindable fishing gear, which is an interesting idea, but not what we're going for. 

Something like:

Fishing gear which is, unintentionally, no longer under the control of or maintained by its original custodian.